tests/BandHQ.test.jsx

```
@@ -45,7 +46,9 @@ mock.module('../src/ui/shared/index.jsx', {
   namedExports: {
     StatBox: () => React.createElement('div', { 'data-testid': 'stat-box' }),
     ProgressBar: () => React.createElement('div', { 'data-testid': 'progress-bar' }),
-    SettingsPanel: () => React.createElement('div', { 'data-testid': 'settings-panel' })
+    SettingsPanel: () => React.createElement('div', { 'data-testid': 'settings-panel' }),
+    Panel: ({ children }) => React.createElement('div', { 'data-testid': 'panel' }, children),
+    ActionButton: ({ children }) => React.createElement('button', { 'data-testid': 'action-button' }, children)
```

Biome (2.4.4)

[error] Provide a valid type prop for the button element.

(lint/a11y/useButtonType)